darts/tests/explainability/test_shap_explainer.py

```diff
@@ -414,7 +410,7 @@ def test_explain_with_lags_future_covariates_series_extending_into_future(self):
         model = LightGBMModel(
             lags=4,
             lags_past_covariates=[-1, -2, -3],
-            lags_future_covariates=[2],
+            lags_future_covariates=[-2],
```

Comment:
I guess this test was expecting a positive lags for the future covariates (extending into the future)